### DISCUSSION TITLE
How to build and install custom version

### DISCUSSION THREAD

**Discussion by baptistesa** [coder/code-server#2156] [Q&A]
Hello,

I discovered your project (good work !) and I'm trying to make some modifications (for example I don't want to see the _Open_) option.
After making these modifications, how we can compile and run our custom version of code-server ?

I tried with _npm run-script build_ , but has several errors (unknown compiler option, after no modifications done). 

Thanks a lot

**Comment by sr229** [comment]
Can you detail what errors you get? If you're getting errors like missing libraries, then you'll need the VS Code dependencies firsthand before attempting to compile code-server.
